### ISSUE TITLE
Decorated functions with no arguments trigger a warning

### ISSUE THREAD

**Issue by adamdelman** [Intsights/flake8-intsights#2]
The following code triggers the following incorrect warning: I061 function definition with no arguments should be defined as: "def function_name():"
```python
@decorator_func
def example():
    print('stuff')
```
